client/js/_author/components/bank_navigation/bank_list.jsx

```diff
@@ -2,8 +2,30 @@ import React      from 'react';
 import _          from 'lodash';
 import Header     from './bank_list_header';
 import ListItem   from './bank_list_item';
+import Spinner    from '../common/spinner';
 
 export default function bankList(props) {
+
```

Comment:
Update spec for this file?  spec exists ? Should be broken from refactor : needs one anyway